pom.xml

```diff
@@ -452,31 +457,16 @@ It supports test-case and test-suite which are Selenium IDE's native format.</de
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <skipTests>${skipTests}</skipTests>
+        	<parallel>classes</parallel>
+        	<useUnlimitedThreads>true</useUnlimitedThreads>
+
         </configuration>
       </plugin>
     </plugins>
   </build>
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>org.seleniumhq.selenium</groupId>
-        <artifactId>selenium-java</artifactId>
-        <version>${selenium.version}</version>
```

Comment:
Please do not delete this part and `</dependency>`.
If you remove this part, the version of selenium-java cannot be determined and a build error will occur.